packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -46,6 +46,7 @@ const ossApi = apiClient(apiURL);
 function replaceIDs(fhirBundle: FHIRBundle, patientId: string): FHIRBundle {
   const stringsToReplace: { old: string; new: string }[] = [];
   for (const bundleEntry of fhirBundle.entry) {
+    if (!bundleEntry.resource) throw new Error(`Missing resource`);
```

Comment:
Failing more gracefully when the resource is missing in the Bundle - [context](https://metriport.slack.com/archives/C04T256DQPQ/p1713715216631559)